src/components/Account/ScreenPromptingAction.js

```diff
@@ -88,32 +83,33 @@ const AccountModuleConnectingScreen = React.memo(function({
             font-weight: 600;
           `}
         >
-          Connecting to {provider.name}
+          {actionTitle}
         </h1>
         <p
           css={`
             width: ${36 * GU}px;
             color: ${theme.surfaceContentSecondary};
           `}
         >
-          Log into {getProviderString('your Ethereum provider', provider.id)}.
-          You may be temporarily redirected to a new screen.
+          {actionDescription}
         </p>
       </div>
       <div
         css={`
           flex-grow: 0;
         `}
       >
-        <Link onClick={onCancel}>Cancel</Link>
+        {onCancel && <Link onClick={onCancel}>Cancel</Link>}
       </div>
     </section>
   )
 })
 
-AccountModuleConnectingScreen.propTypes = {
-  providerId: PropTypes.string,
-  onCancel: PropTypes.func.isRequired,
+AccountModuleActionScreen.propTypes = {
+  actionTitle: PropTypes.string,
+  actionDescription: PropTypes.string,
+  logo: PropTypes.img,
```

Comment:
This is throwing an error because there is no `img`  prop type. When importing images you get a string which indicates the path to that file. I'd replace this with `Proptypes.string`